src/cli.rs

```diff
@@ -248,6 +248,7 @@ pub async fn cli() -> Result<(), Box<dyn Error>> {
             whole_stream_command(Next),
             whole_stream_command(Previous),
             whole_stream_command(Debug),
+            // whole_stream_command(Docker),
```

Comment:
You can just remove this line

Reply:
I thought I had removed the commented out code, sorry about that.